purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -139,6 +139,12 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             appConfig.finishTransactions = value
         }
 
+    /**
+     * Exposes the observerMode configuration option
+     */
+    val observerMode: Boolean
```

Comment:
could've just used finishTransactions, but i know we are deprecating that soon. this one is read-only

Reply:
I think this should be internal so no one changes it after configuring

Reply:
if we make it internal the amazon package can't access it (there's probably a way around that though?)...it's a val though so they shouldn't be able to change it.

Reply:
oh that's true my bad, for some reason I read it as a var, let's keep it like this then!